File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ChangeSettingResponse.java
package org.fossasia.susi.ai.rest.responses.susi;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by meeera on 30/6/17.
 */

public class ChangeSettingResponse {
    @SerializedName("session")
    @Expose
    private Session session;
    @SerializedName("message")
    @Expose
    private String message;

    public Session getSession() {
        return session;
    }

    public String getMessage() {
        return message;
    }
}

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/ChangeSettingResponse.kt
package org.fossasia.susi.ai.rest.responses.susi


/**
 * Created by meeera on 30/6/17.
 */

data class ChangeSettingResponse(
        /**
         * Gets session
         *
         * @return the session
         */
        val session: Session? = null,

        /**
         * Gets message
         *
         * @return the message
         */
        val message: String? = null
)
